src/user/auth.js

```diff
@@ -107,9 +107,18 @@ module.exports = function (User) {
 			return;
 		}
 		await cleanExpiredSessions(uid);
+		await revokeSessionsAboveThreshold(uid, meta.config.maxUserSessions || 10);
```

Comment:
Instead of setting fallback value here, include it in `install/defaults.json`

Reply:
Will it be sufficient or I need it in two places